bindings/rust/s2n-tls-tokio/tests/handshake.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_tls::raw::{config::Config, error::Error, security::DEFAULT_TLS13};
+use s2n_tls_tokio::{TlsAcceptor, TlsConnector};
+use tokio::net::{TcpListener, TcpStream};
+
```

Comment:
```suggestion
/// NOTE: this certificate and key are to be used for demonstration purposes only!
```

Reply:
I don't want to live in a world where this comment is necessary on a test D: